hyperapp.d.ts

```diff
@@ -21,6 +21,11 @@ export interface Component<Attributes = {}> {
   (attributes: Attributes, children: Array<VNode | string>): VNode<Attributes>
 }
 
+/**
+ * Possibles children types
+ */
+export type Children = VNode | string | number | null;
```

Comment:
..and function (subview) #639

Reply:
I think that belongs to the definition of VNode.

Reply:
Why then `string`, `number` and `null` are not VNode definitions?

Virtual Tree:
```js
{ // VNode
  key,
  nodeName,
  attributes,
  children: [
    { key, nodeName, attributes, children }, // VNode
    "string", // text node
    123, // text node
    null, // currently not supported
    undefined, // currently not supported
    () => {}, // subview (just function)
    () => ({ key, nodeName, attributes, children }), // subview may return VNode
    () => "string", // subview may return text node
    () => 123, // subview may return text node
    () => null, // currently not supported
    () => undefined, // currently not supported
  ]
}
```